install_dependencies.sh

```diff
@@ -39,7 +39,7 @@ if [ -f /etc/lsb-release ]; then
             ca-certificates \
             git \
             libboost-regex-dev \
-            $x86_64_specific_packages \
+            "$x86_64_specific_packages" \
```

Comment:
This looks like a mistake - `x86_64_specific_packages` contains multiple names so you can't pass them as one argument. The way to fix it would be to use an array.

Reply:
reverted